internal/services/redis/redis_cache_resource.go

```diff
@@ -376,6 +382,21 @@ func resourceRedisCache() *pluginsdk.Resource {
 				}
 				return false
 			}),
+			pluginsdk.CustomizeDiffShim(func(ctx context.Context, diff *pluginsdk.ResourceDiff, v interface{}) error {
+				// Entra (AD) auth has to be set to disable access keys auth
+				// https://learn.microsoft.com/en-us/azure/azure-cache-for-redis/cache-azure-active-directory-for-authentication
+
+				accessKeysAuthenticationDisabled := !(diff.Get("access_keys_authentication_enabled").(bool))
```

Comment:
Can we also invert the local variable here, to use `accessKeysAuthenticationEnabled` instead?

Reply:
Implemented in faa348a642